src/util/ChartUtils.ts

```diff
@@ -1337,3 +1338,5 @@ export const getTooltipItem = (graphicalItem: ReactElement, payload: any) => {
     chartType,
   };
 };
+
+export const isAxisLTR = (axisMap: XAxisMap) => Object.values(axisMap ?? {}).some(({ reversed }) => !reversed);
```

Comment:
Can we have a unit test for this new function? Should be easy - true, false, undefined, expected results.

Reply:
Added unit tests